packages/amplify_datastore_plugin_interface/lib/src/types/query/query_field.dart

```diff
@@ -24,8 +27,9 @@ part 'query_sort.dart';
 
 class QueryField {
   final String fieldName;
+  final ModelFieldType fieldType;
 
-  const QueryField(this.fieldName);
+  const QueryField({@required this.fieldName, this.fieldType});

```

Comment:
Will this impact the queryPredicate generation DX?

Reply:
I added this for our own internal usage so it won't affect the DX